doc-examples/hibernate-example-java/src/main/java/example/ProductManager.java

```diff
@@ -1,20 +1,19 @@
 
 package example;
 
-import io.micronaut.transaction.SynchronousTransactionManager;
+import io.micronaut.transaction.TransactionOperations;
 import jakarta.inject.Singleton;
 import jakarta.persistence.EntityManager;
-import java.sql.Connection;
+import org.hibernate.Session;
 
 @Singleton
 public class ProductManager {
 
     private final EntityManager entityManager;
-    private final SynchronousTransactionManager<Connection> transactionManager;
+    private final TransactionOperations<Session> transactionManager;
 
```

Comment:
need to add more documentation to `breaks` about this change and how programmatic transactions have changed

Reply:
The previous way is still supported (`SynchronousTransactionManager<Session>`) but I would like to eliminate `SynchronousTransactionManager` because of the detached TX is not the best for propagation